scripts/bravo_csv.py

```diff
@@ -698,30 +706,60 @@ def calc_vol(art_tuple, logContext, checkTheLog):
             org_vol = art_tuple[0]['uri'].udf['Volume (ul)']
         volume = float(amount_ng) / float(conc)
 
+        final_volume = art_tuple[1]['uri'].udf["Total Volume (uL)"]
+
         if org_vol < MIN_WARNING_VOLUME:
             logContext.write("WARN : Sample {0} located {1} {2}  has a LOW original volume : {3}\n".format(art_tuple[1]['uri'].samples[0].name,
-                                                                                                  art_tuple[0]['uri'].location[0].name, art_tuple[0]['uri'].location[1], org_vol))
+                                                                                                           art_tuple[0]['uri'].location[0].name,
+                                                                                                           art_tuple[0]['uri'].location[1],
+                                                                                                           "{0:.2f}".format(org_vol)))
             volume = min(org_vol, volume)
             checkTheLog[0] = True
         elif volume < MIN_WARNING_VOLUME:
-            # arbitrarily determined by Sverker Lundin
-            logContext.write("WARN : Sample {0} located {1} {2}  has a LOW pippetting volume: {3}\n".format(art_tuple[1]['uri'].samples[0].name,
-                                                                                                  art_tuple[0]['uri'].location[0].name, art_tuple[0]['uri'].location[1], volume))
+            # When the volume is lower than the MIN_WARNING_VOLUME, set volume to MIN_WARNING_VOLUME
```

Comment:
Did the old comment say that the MIN_WARNING_VOLUME is an arbitrary value? 😅

Reply:
Yes, I guess it was some number before :D